src/components/scenes/TransactionDetailsScene.js

```diff
@@ -182,7 +182,12 @@ export class TransactionDetailsComponent extends React.Component<Props, State> {
         amount={this.state.amountFiat}
         onChange={this.onChangeFiat}
       />
-    )).then(_ => {})
+    )).then(_ => {
+      const { amountFiat } = this.state
```

Comment:
Use this util function to check if a string number is zero: 
https://github.com/EdgeApp/edge-react-gui/blob/develop/src/util/utils.js#L149